crowbar_framework/config/environments/development.rb

```diff
@@ -51,9 +51,7 @@
   config.log_level = :debug
   config.log_tags = []
 
-  config.logger = ActiveSupport::TaggedLogging.new(
-    Logger.new Rails.root.join("log", "development.log") # SyslogLogger.new
-  )
-
-  config.log_formatter = ::Logger::Formatter.new
+  logger = ActiveSupport::Logger.new(Rails.root.join("log", "development.log"))
+  logger.formatter = ::Logger::Formatter.new
```

Comment:
```Logger::Formatter``` won't work as it doesnt respect the tags. I think we have to create a formatter within lib folder and set it here.